Makefile

```diff
@@ -20,6 +20,17 @@ py_lint: # Check Python code with isort, black and pylint to identify any proble
 	${PYTHON} -m black --line-length ${LINE_LENGTH} --check ${PROJECT_NAME}/* *.py
 	${PYTHON} -m pylint ${PROJECT_NAME}/*
 
+js_setup: # Setup dependencies for JS code
+	mkdir -p ${GITHUB_PAGES_DIR}/assets/javascript/vendor/
+	cd ${GITHUB_PAGES_DIR} && \
+		curl -X GET https://cdn.plot.ly/plotly-2.12.1.min.js \
+		> assets/javascript/vendor/plotly.js
```

Comment:
`plotly.js` is not needed anymore

```suggestion
```

Reply:
Done https://github.com/Doreapp/prix-carburants/pull/36/commits/da3f93a69c679ce7394c1b4661a4eb9da6ef49d1